src/components/cv/kompetanser.tsx

```diff
@@ -1,22 +1,23 @@
-import { BodyShort } from '@navikt/ds-react';
+import { BodyShort, List } from '@navikt/ds-react';
 import { Jobbprofil } from '../../data/api/datatyper/arenaperson';
 import EMDASH from '../../utils/emdash';
 import Informasjonsbolk from '../felles/informasjonsbolk';
 import { FolderFileIcon } from '@navikt/aksel-icons';
+import ListItem from '@navikt/ds-react/esm/list/ListItem';
 
 const Kompetanser = ({ kompetanse }: Pick<Jobbprofil, 'kompetanse'>) => {
     const kompetanser =
         kompetanse && kompetanse.length > 0
             ? kompetanse?.map((kompetansen) => (
-                  <li>
+                  <ListItem>
                       <BodyShort size="small">{kompetansen.tittel}</BodyShort>
-                  </li>
+                  </ListItem>
               ))
             : EMDASH;
 
     return (
         <Informasjonsbolk header="Kompetanser" icon={<FolderFileIcon />} headerTypo="ingress">
-            <ul> {kompetanser}</ul>
+            <List>{kompetanser}</List>
```

Comment:
Ikke spesielt viktig men jeg la bare merke til at `ul` (altså "unordered list") er defaulten som blir satt for `List`-komponenten, men det er ikke noe i vegen for å være eksplisitt, altså at vi sier:

 ```suggestion
            <List as="ul">{kompetanser}</List>
```

da blir det litt tydeligere for de som leser koden at det er en `ul`-liste 😄